src/components/Headshots/Headshot.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { faInstagram, faLinkedin, faTiktok, faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { faWindowMaximize } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon as Icon } from "@fortawesome/react-fontawesome";
+import "./Headshot.css";
+
+const Headshot = ({ team, name, role, insta, linkedin, website }) => {
+	const path = `./headshots/${team}/${name}.webp`;
+	const alt = `image of ${name}`;
+	role = role ?? "";
+
+	let instaDiv;
+	if (insta) {
+		instaDiv = (
+			<a href={`https://www.instagram.com/${insta}`} target="_blank" rel="noreferrer" aria-label="Instagram">
+				<Icon className="invert" icon={faInstagram} />
+			</a>
+		);
+	} else {
+		instaDiv = <div></div>;
+	}
```

Comment:
Why do we want an empty `<div>` if `insta` is false?
I would just write:

```jsx
const instagramLink = instagram ? (
	<a href={`https://www.instagram.com/${instagram}`} target="_blank" rel="noreferrer" aria-label="Instagram">
		<Icon className="invert" icon={faInstagram} />
	</a>
) : (
	<></>
);
```
(renaming the variable to `instagram` and the link to `instagramLink` while using a JSX fragment instead of an empty `<div>`)